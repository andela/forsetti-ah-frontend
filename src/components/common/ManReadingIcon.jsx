import React from 'react';
import manreadicon from '../../assets/svg/manreadicon.svg';

const ManReadingIcon = ({ className }) => (
  <div className={className}>
    <img src={manreadicon} alt='Man reading icon' />
  </div>
);

export default ManReadingIcon;
