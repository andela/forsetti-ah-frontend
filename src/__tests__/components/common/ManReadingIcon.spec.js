import React from 'react';
import { shallow } from 'enzyme';
import ManReadingIcon from '../../../components/common/ManReadingIcon';

describe('Man Reading Icon ', () => {
  it('should render ManReadingIcon correctly', () => {
      const wrapper = shallow(<ManReadingIcon />);

      expect(wrapper).toMatchSnapshot();
  });
});
