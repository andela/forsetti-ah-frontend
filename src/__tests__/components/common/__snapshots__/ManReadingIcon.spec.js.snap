// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Man Reading Icon  should render ManReadingIcon correctly 1`] = `ShallowWrapper {}`;
